sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/annotation/ReturnType.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.generic.core.annotation;
+
+/**
+ * Enumeration of return types used with {@link ServiceMethod} annotation to indicate if a method is expected to return
+ * a single item or a collection.
+ */
+public enum ReturnType {
```

Comment:
do we not need any rest proxy impl c changes o handle the return type?

Reply:
From having a look at the Azure Core counterpart, no. It seems this is only for static code analysis tools like CheckStyle and for when we use APIView.

Reply:
Maybe @JonathanGiles or @srnagar have a different point of view.